docs/reST/ref/math.rst

```diff
@@ -60,6 +60,10 @@ Multiple coordinates can be set using slices or swizzling
 
    Some general information about the ``Vector2`` class.
 
+   .. versionadded:: 2.1.3
+      The Vector2 class can be subclassed. Methods such as ``copy()`` and ``move_towards()``
+      will recognize this and return instances of the subclass.
```

Comment:
vectors were always subclassable. What this PR changes is the return types of inherited methods to match the subclass